docs/applications/pyclient/components-overview.rst

```diff
@@ -1,3 +1,5 @@
+.. _client-comp-overview:
```

Comment:
if you **don't** want to link to a specific section in a doc, I would rather not define unique refs and then link to them.
To link to whole documents directly, please use the `:doc:` directive (as explained [here](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-doc) )